src/content/enterprise/administration/configuration.mdx

```diff
@@ -287,11 +288,30 @@ autoscaler:
   slackWebhook:
 ```
 
+### backend
+
+Common configuration for backend services.
+
+- `pipelineInstaller.image`: Base image for the binaries needed by the [installer jobs](/docs/enterprise/administration/configuration/#installer).
+- `installer.annotations`: Annotations to add to the [installer jobs](/docs/enterprise/administration/configuration/#installer).
+- `installer.env`: Environment variables to add to the [installer jobs](/docs/enterprise/administration/configuration/#installer).
+- `installer.image`: Custom runtime image for the [installer jobs](/docs/enterprise/administration/configuration/#installer). Defaults to [okteto/pipeline-runner](https://github.com/okteto/pipeline-runner).
+- `installer.labels`: Labels to add to the [installer jobs](/docs/enterprise/administration/configuration/#installer).
+- `image.registry`: Image registry URL for the [api](/docs/enterprise/administration/configuration/#api) and [webhook](/docs/enterprise/administration/configuration/#webhook) services.
+- `image.tag`: Image tag for the [api](/docs/enterprise/administration/configuration/#api) and [webhook](/docs/enterprise/administration/configuration/#webhook) services.
+
+```yaml
+backend:
+  installer:
+    image: okteto/pipeline-runner:1.0.0
+```
+
 ### buildkit
 
 The build service. It's used in combination with `okteto build` to build containers directly in the cluster.
 
 - `annotations`: Annotations to add to the buildkit pods.
+- `env`: Environment variables to add to the buildkit pods.
```

Comment:
This change suggestion applies to all the other ones as well, since technically the env var is applied to the container and not to the entire pod.
```suggestion
- `env`: Environment variables to be set on the buildkit containers.
```